match2/commons/match_group_display_matchlist.lua

```diff
@@ -136,8 +142,9 @@ function MatchlistDisplay.Match(props)
 
 	-- Compute widths of the 2 opponent and 2 score columns. The small offsets
 	-- are due to border splitting from table-layout: auto.
-	local opponentWidth = math.floor(0.4 * props.width) - 1
-	local scoreWidth = 0.5 * (props.width - 5 - 2 * opponentWidth)
+	local minScoreWidth = 30
+	local scoreWidth = math.max(minScoreWidth, math.floor(0.1 * props.width) - 1)
```

Comment:
Nice, `minScoreWidth` is much, much clearer. Make it a constant in the file though! [See style guide](https://liquipedia.net/commons/Help:Scribunto#Identifiers). So:

```suggestion
	local scoreWidth = math.max(_MIN_SCORE_WIDTH, math.floor(0.1 * props.width) - 1)
```

Reply:
> Nice, `minScoreWidth` is much, much clearer. Make it a constant in the file though! [See style guide](https://liquipedia.net/commons/Help:Scribunto#Identifiers). So:

I'm not a fan of using all caps for variable names - mostly because constants aren't interesting enough have its identifier distinguished, and because when programming in a functional style nearly everything is a constant. In this case I also don't want to move `local minScoreWidth` to the top of the file, because it would be too far away from where it's used.

Reply:
I'm not going to accept this PR till it abides by the style guide. If you disagree with the style guide, you can raise that on Discord, but this PR needs to abide by it. 

> I also don't want to move local minScoreWidth to the top of the file, because it would be too far away from where it's used.

We have intellisense for that.

Reply:
> I'm not going to accept this PR till it abides by the style guide. If you disagree with the style guide, you can raise that on Discord, but this PR needs to abide by it.

The style guide only requires global variables (*) to be in all caps. So if `local minScoreWidth` is kept local then it should be fine right?

(*) I'm assuming only those variables of a certain kind, since it doesn't make sense to have `local Logic = require('Module:Logic')` be in caps

Reply:
Sure, you're right. This PR gets a green tick then, but I'm gonna change the style guide 😉